sample-skill/lambda/lambda_function.py

```diff
@@ -117,6 +118,25 @@ def get_course_id(course_name: str) -> str | None:
     return COURSES.get(course_name)
 
 
+def get_profile_email(handler_input: HandlerInput) -> str | None:
+    """Return the email of the user.
+
+    Args:
+        handler_input (HandlerInput): Handler input
+
+    Returns:
+        str: Email of the user
+        None: If the email can't be retrieved
+    """
+    ups_service_client = handler_input.service_client_factory.get_ups_service()
+    return ups_service_client.get_profile_email()
+
+
+def get_username_by_profile_email(_: str) -> str:
+    """Return the username of the user by email."""
+    return "student"
```

Comment:
why is this hardcoded?

Reply:
@mariajgrimaldi, I removed the hardcoded code and added the logic to obtain the username by consuming the eox-core endpoint